Makefile

```diff
@@ -196,3 +186,6 @@ fmt:
 .PHONY: lint
 lint: $(gobin_tool)
 	gobin -run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.33.0 run ./...
+
+gotestsum: $(gobin_tool)
+	gobin gotest.tools/gotestsum@v1.6.4
```

Comment:
shouldn't it just be:
`gobin -run gotest.tools/gotestsum@v1.6.4 -- ` -- versus installing it into the path?

Reply:
Then I have to re-execute the Makefile to run gotestsum.